File: PYTHON/2.py
#!/usr/bin/env python
# coding: utf-8

# In[9]:


arr = [25, 11, 7, 75, 56];     
     
#Initialize max with first element of array.    
max = arr[0];    
     
#Loop through the array    
for i in range(0, len(arr)):    
    #Compare elements of array with max    
   if(arr[i] > max):    
       max = arr[i];    
print("Largest element present in given array: " + str(max));  
min = arr[0];    
     
#Loop through the array    
for i in range(0, len(arr)):    
    #Compare elements of array with min    
   if(arr[i] < min):    
       min = arr[i];    
     
print("Smallest element present in given array: " + str(min));    


# In[ ]:





# In[ ]:

File: Java/2.java

//Maximum and Minimum Element
import java.io.*;

class MixMin {

    static int getMin(int arr[], int n) {
        int r = arr[0];

        for (int i = 1; i < n; i++)
            r = Math.min(res, arr[i]);
        return r;
    }

    static int getMax(int arr[], int n) {
        int r = arr[0];

        for (int i = 1; i < n; i++)
            r = Math.max(res, arr[i]);
        return r;
    }

    public static void main(String[] args) {
        int arr[] = {1, 2, 3, 4, 5, 6, 7, 8, 9};
        int n = arr.length;
        System.out.println("Minimum element" + " of array: " + getMin(arr, n));
        System.out.println("Maximum element" + " of array: " + getMax(arr, n));
    }
}
